### PR TITLE
[6.8] bump version

### PR DESCRIPTION
**Pull request by mdelapenya**
## What does this PR do?
It bumps Beats version.

On CI, it also allows forcing a specific (an older) version of python for the Windows workers.

## Why is it important?
CI builds on windows are failing (See #5324)

## Related issues
- Supersedes #5324

### CODE REVIEW

File: script/jenkins/windows-test.ps1 [hunk 1]
```diff
@@ -28,7 +28,7 @@ $env:PATH = "$env:GOPATH\bin;C:\tools\mingw64\bin;$env:PATH"
 $env:MAGEFILE_CACHE = "$env:WORKSPACE\.magefile"
 
 # Setup Python.
-exec { choco install python2 -y -r --no-progress --version 2.7.17 }
+exec { choco install python2 -y -r --force --no-progress --version 2.7.17 }
```
Comment (mdelapenya) [inline script/jenkins/windows-test.ps1:31]: OTOH, Python 2.7.18 is installed in the workers. We could simply remove the install command and use workers' Python. Thoughts?

**Review by cachedout — APPROVED** [review]

File: script/jenkins/windows-test.ps1 (hunk 1 above)
Comment (v1v) [reply script/jenkins/windows-test.ps1:31]: I guess, we could do so

**Review by v1v — APPROVED** [review]

File: script/jenkins/windows-test.ps1 (hunk 1 above)
Comment (mdelapenya) [reply script/jenkins/windows-test.ps1:31]: Will wait for the @elastic/apm-server team for clarification about the desired Python version. Will remove the installation  if they agree.

Comment (jalvz) [reply script/jenkins/windows-test.ps1:31]: that should be fine, but going to merge this nevertheless 👍

**Review by jalvz — APPROVED** [review]

**Pull request merged**